src/main/java/com/chariotsolutions/meetupnotifier/EventIdRow.java

```diff
@@ -0,0 +1,20 @@
+package com.chariotsolutions.meetupnotifier;
+
+public class EventIdRow {
+  private String googleId;
+  private String meetupId;
+
+  public EventIdRow(String googleId, String meetupId) {
+    this.googleId = googleId;
```

Comment:
Yeah there's no need to use Derby, but thought you may be interested in knowing about it.  I don't know if I'd user either for serious production work, but fine for things like this.